swc/private/swc.bzl

```diff
@@ -121,16 +130,16 @@ def _calculate_js_out(src, out_dir = None, root_dir = None, js_outs = []):
         ".cjs": ".cjs",
         ".cts": ".cjs",
     }
-    js_out = paths.replace_extension(src, _replace_ext(src, exts))
+    js_out = _replace_extension(src, _replace_ext(src, exts))
```

Comment:
could you file the issue on bazel_skylib so we know why we have to workaround their perf bug?

Reply:
https://github.com/bazelbuild/bazel-skylib/issues/487

Reply:
cool, we could consider making a fixed version in aspect-bazel-lib but let's give upstream a chance to care first...